LoopKitUI/HorizontalSizeClassOverride.swift

```diff
@@ -20,4 +20,12 @@ public extension HorizontalSizeClassOverride {
             return .regular
         }
     }
+        
+    var isDisplayNarrow: Bool {
+        if UIScreen.main.bounds.width < 375 {
```

Comment:
[nit] It would make me get more warm fuzzies if this (and the 640 above) were constants named for what iPhone size they represent.